iree/samples/variables_and_state/variables_and_state.ipynb

```diff
@@ -0,0 +1,471 @@
+            "#loc7 = loc(callsite(\"AssignVariableOp@__inference_set_value_23\"(\"/usr/local/lib/python3.7/dist-packages/tensorflow/python/ops/gen_resource_variable_ops.py\":155:0) at callsite(\"/usr/local/lib/python3.7/dist-packages/tensorflow/python/ops/resource_variable_ops.py\":903:0 at callsite(\"<ipython-input-3-719deffc4cc3>\":14:0 at callsite(\"/usr/local/lib/python3.7/dist-packages/tensorflow/python/framework/func_graph.py\":982:0 at callsite(\"/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/function.py\":3971:0 at callsite(\"/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/def_function.py\":672:0 at callsite(\"/usr/local/lib/python3.7/dist-packages/tensorflow/python/framework/func_graph.py\":999:0 at callsite(\"/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/function.py\":3289:0 at callsite(\"/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/function.py\":3444:0 at \"/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/function.py\":3050:0))))))))))\n",
+            "\n",
+            " ```\n",
+            "\n",
+            "Wrote MLIR to path '/tmp/iree/colab_artifacts/counter.mlir'\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "WCiRV6KRh3iA"
+      },
+      "source": [
+        "## Test the imported program\n",
+        "\n",
+        "_Note: you can stop after each step and use intermediate outputs with other tools outside of Colab._\n",
+        "\n",
+        "* _The \"imported MLIR\" can be used by IREE's generic compiler tools that run on Linux, macOS, and Windows_\n",
```

Comment:
would be nice to include a minimal example command line or link to documentation about it - otherwise this just like "ok cool cool but now how do I do that?" :P

Reply:
The README.md file has a `Changing compilation options` section that goes over this. I can add a reference to that.

Reply:
you can use an anchor to link right to that section on github, too:
https://gist.github.com/rachelhyman/b1f109155c9dafffe618